build.sbt

```diff
@@ -3189,10 +3191,17 @@ lazy val `runtime-version-manager` = project
   .dependsOn(pkg)
   .dependsOn(downloader)
   .dependsOn(cli)
+  .dependsOn(`process-utils`)
   .dependsOn(`version-output`)
   .dependsOn(`edition-updater`)
   .dependsOn(`distribution-manager`)
 
+lazy val `process-utils` = project
+  .in(file("lib/scala/process-utils"))
```

Comment:
Are we planning to use this code in production or just in tests? If it is intended for production, I'd rather place the project to `lib/java` and write it in Java.

Reply:
The PR **moves** the file from tests to regular code. It's a plain utility code, please don't ask me to rewrite this to Java at this point.